src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -811,13 +828,21 @@ private PreparedStatement getDocFromDb(Connection conn,
       String uniqueId) throws SQLException {
     PreparedStatement st = conn.prepareStatement(singleDocContentSql);
     uniqueKey.setContentSqlValues(st, uniqueId);  
+    if (aclSql == null) {
+      uniqueKey.setAclSqlValues(st, uniqueId);
```

Comment:
This seems wrong. The db.aclSqlParameters columns are specific to db.aclSql, and may not be the same in db.singleDocContentSql. Did something fail without this?

Reply:
Since this is wrong, too, maybe add a test where doing this would fail, too.

Reply:
Deleted the newly added lines, as these are not required. And test required.